src/app/static/src/js/components/runReport/parameterList.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="col-sm-6">
+      <table class="table table-sm table-bordered">
+        <tbody>
+        <tr v-for="(param, index) in params" :key="index">
+          <th><label :for="`param-control-${index}`"
```

Comment:
Is there any reason why these are `<th>` rather than `<td>`?

Reply:
I have made label th and input is td.

Reply:
Oh ok so you're making use of the default th styling for the label cells?